pom.xml

```diff
@@ -170,6 +170,14 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-resources-plugin</artifactId>
+				<version>3.3.1</version>
+				<configuration>
+					<propertiesEncoding>ISO-8859-1</propertiesEncoding><!-- To avoid warning during plugin execution. The project currently doesn't include properties files. -->
```

Comment:
Wouldn't it be better to set it to UTF-8?

By the way do you know why it's not already the default?

Reply:
As the project currently target Java 8 I choose to use `ISO-8859-1` over `UTF-8`. But as the  LSC doesn't include any properties file the configuration here does not actually matters and it safe to set the value to `UTF-8`.

Reply:
I totally agree with you and change the encoding from `ISO-8859-1` to `UTF-8` in a new commit.

I don't know how but I didn't find the properties files during my first scan of the project files, sorry.

Reply:
No problem, than you very much for the change!